addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -714,14 +714,16 @@ export class OdooEditor extends EventTarget {
 
     sanitize(target) {
         this.observerFlush();
-
         let record;
         if (!target) {
             // If the target is not given,
             // find the closest common ancestor to all the nodes referenced
             // in the mutations from the last step.
             for (record of this._currentStep.mutations) {
                 const node = this.idFind(record.parentId || record.id) || this.editable;
+                if (!this.editable.contains(node)) {
+                    continue;
+                }
```

Comment:
Shouldn't the right fix be to find the _current_ parent instead of the old one when the old one is not in the DOM anymore ? It seems weird to just entirely _ignore_ that particular mutated node from the computation if it is still in the DOM and it's only its previous parent that is not in the DOM anymore. I think it should be something like this (in pseudo code):
```js
if (mutated node still in the dom) {
    parent  = node.oldParent still in the dom ? node.oldParent : node.newParent
    // ... continue computing common ancestor
}
```

Reply:
Well we don't relay care if it's still in the DOM right ? We just care if he is still somewhere in the editable element.

Reply:
That's what I meant, sorry for not being clear.

Reply:
ok.
Then I don't understand what you want me to change, since it's what I check.